malcolm/modules/pmac/parts/pmacstatuspart.py

```diff
@@ -23,3 +26,14 @@ def servo_frequency(self, context):
         freq = 8388608000. / child.i10.value
         return freq
 
+    @add_call_types
+    def report_status(self, context):
```

Comment:
I don't think anything actually consumes this info, so you can delete all the changes in this file